easybuild/easyconfigs/r/Ruby-FPM/Ruby-FPM-2.1.5-1.0.0-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-FPM'
```

Comment:
It sounds like a good idea.

I do agree that they should be grouped in some way, either under the main language directory, or in some in some other directory either beside (like Ruby-Bundles) or under (like Ruby/Ruby-Bundles/<bunch of ebs>).

I'll test what you suggest as Ruby is really a light package, but for R, Python, Perl we probably want to think about this a bit

@boegel feedback?